src/main/test/audio/AudioTest.java

```diff
@@ -1,26 +1,33 @@
 package audio;
 
 import org.junit.Test;
-import util.Assets;
 
 public class AudioTest {
     private static AudioSource s;
     private static AudioListener l;
-    private static AudioBuffer b;
-
 
     @Test
-    public void play() {
+    public void playTwoAudioFiles() {
         AudioMaster.get();
         l = AudioListener.get();
         l.start();
-        b = Assets.getAudioBuffer("src/assets/sounds/test1.wav");
-        System.out.println(b.getAudioData().length);
-        s = new AudioSource(b);
+        s = new AudioSource("src/assets/sounds/test1.wav", "src/assets/sounds/test2.wav");
         s.start();
+        AudioSource s1 = new AudioSource("src/assets/sounds/test1.wav");
+        s1.start();
         System.out.println("playing");
-        s.play(0);
+        s.play(1);
+        // note: playing two noises from the same AudioSource before the first one finishes
+        // stops the first one from finishing.
+        s1.play(0);
+
 
+        try {
+            // simulate things happening in game
+            Thread.sleep(10000);
```

Comment:
I suggest to never do this in a test - a test is never supposed to be running longer than the code takes - artifical waiting time slows down the compilation process, which will get slower over time anyways.

Reply:
You're right, I'll probably make another scene as a test instead of this when I can get to it.

Reply:
yes, manual tests are preferable for OpenAL as well as everything related with OpenGL and so on